templates/locales/en.yml

```diff
@@ -877,6 +877,14 @@ en:
         the issues below and execute "vagrant reload":
 
         %{output}
+      nfs_exports_failed: |-
+        Vagrant failed to install an updated NFS exports file. This may be
+        due to overly restrictive permissions on your NFS exports file. Please
+        validate them and try again.
```

Comment:
Hi! You make a good point on the content reading, which I will update to shell out being consistent with the `chown` and `mv`. The edge case this error is mainly in use for would be where the file is detected as readable or accessible to the user (and sudo is thus not used) and the permissions on the file is changed between the check and the actual operation (like the `mv`).